html/package.json

```diff
@@ -35,22 +35,20 @@
   "types": "lib/cjs/index.d.ts",
   "dependencies": {
     "@cycle/run": "*",
-    "@cycle/dom": "18.x",
-    "snabbdom": "0.6.9",
-    "snabbdom-to-html": "3.1.1"
-  },
-  "peerDependencies": {
+    "@cycle/dom": ">=19.3.0",
+    "snabbdom": "0.7.0",
+    "snabbdom-to-html": "3.3.0",
     "xstream": "*"
   },
   "devDependencies": {
-    "@cycle/isolate": "2.1.0",
```

Comment:
Add peer dependencies to devDependencies

Reply:
Which dependencies? I removed peerDependencies because with yarn and npm5, we can use dependency `xstream: "*"` to signal that the parent package is free to specify which xstream version to use, but in the end of the day there must be some xstream version.

The case for cycle/isolate: it's only used in tests.

Reply:
I meant `snabbdom` and `snabbdom-to-html`. They are needed for the tests (or at least should be), so with `peerDependencies` only they wont be installed on `make setup`

Reply:
snabbdom and snabbdom-to-html are both `dependencies` in this PR

Reply:
Oh, I was getting confused with the diff, sorry, all green then